### DISCUSSION TITLE
Why GitHub give me that issue

### DISCUSSION THREAD

**Discussion by YazanAwawda** [community/community#52658] [GitHub Education]
Hey YazanAwawda,

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Apr 12, 2023 for @YazanAwawda.

You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.
The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.
You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

**Comment by YazanAwawda** [comment]
How to resolve that issue

**Reply by samus-aran** [reply]
Hi @YazanAwawda How did you get on? Did you proceed with an additional submission after @AbderraoufAbla's suggestions?

**Comment by opegamanalitarok** [comment]
It seems like the message you received is indicating that there are a few issues with the information you submitted for verification of your academic affiliation on GitHub. Here are the specific issues mentioned:

1. Your GitHub user profile is missing your full name and a short bio. Make sure you have provided this information in your profile.

2. The school you selected does not have a campus location in your country. If you are attending the school remotely, your academic affiliation document should indicate this.

3. GitHub cannot verify your academic affiliation if you are not near any campus location for the school you selected. If you are a distance learner, your academic affiliation documentation must state this.

To resolve these issues, you should update your GitHub user profile to include your full name and a short bio. Additionally, if you are attending your selected school remotely, make sure your academic affiliation document reflects this. If you are a distance learner, ensure that your academic affiliation documentation states this clearly. Once you have updated this information, you can resubmit your verification request to GitHub.

Hope this helps! 🙌🙌

**Reply by rushiiMachine** [reply]
Why do I have to make my full name public in order to get verified? I already supplied my school affiliated email a dated id WITH MY FULL NAME and I STILL get denied because I want to maintain my privacy? What a nice way to get doxxed and swatted
